spec/services/import_actuals_spec.rb

```diff
@@ -188,9 +213,12 @@
       end
     end
 
-    context "when the Value is blank" do
+    context "when the Actual Value and the Refund Value are blank" do
```

Comment:
This test is misleading: the higher-level behaviour is that if these are blank, the row is skipped entirely.